x/wasm/keeper/query_plugins_test.go

```diff
@@ -445,6 +453,40 @@ func TestBankQuerierAllMetadata(t *testing.T) {
 	assert.Equal(t, exp, got)
 }
 
+func TestBankQuerierAllMetadataPagination(t *testing.T) {
+	var requestedPagination *query.PageRequest
```

Comment:
Nit: to make it more verbose
```suggestion
	var capturedPagination *query.PageRequest
```